src/DocumentType.ts

```diff
@@ -0,0 +1,28 @@
+import Node from './Node';
+
+/**
+ * The DocumentType interface represents a Node containing a doctype.
+ */
+export default class DocumentType extends Node {
+	public name: string;
+	public publicId: string;
+	public systemId: string;
+
+    /**
+	 * @param name     The name of the document type
+	 * @param publicId The public identifier of the doctype
+	 * @param systemId The system identifier of the doctype
+	 */
+	constructor (name: string, publicId: string, systemId: string) {
+		super(Node.DOCUMENT_TYPE_NODE);
+
+		this.name = name;
+		this.publicId = publicId;
+		this.systemId = systemId;
+	}
+
+	public cloneNode (deep: boolean = true, _copy?: DocumentType): DocumentType {
```

Comment:
The underscore was intended to indicate private. I agree that it clashes in a confusing way with the convention for unused arguments. I'll remove the underscores for function arguments and just document them as non-standard.